OpenRA.Game/ContentInstaller.cs

```diff
@@ -0,0 +1,39 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA
+{
+	public class ContentInstaller : IGlobalModData
```

Comment:
Could you please add a comment `// Referenced from ModMetadata, so needs to be in OpenRA.Game :(` before this?